### PR TITLE
Expose two ports for pilot envoy

### PR DESCRIPTION
**Pull request by wattli**
Pilot envoy has two new ports: 15007 for http and 15005 for https. This fixes the bug that when pilot ControlPlaneAuthPolicy changes, others won't be able to talk to it.

### CODE REVIEW

**Comment by rkpagadala** [comment]
Thanks Tao

File: install/kubernetes/templates/istio-pilot.yaml.tmpl [hunk 1]
```diff
@@ -157,13 +161,15 @@ spec:
...
+        - containerPort: 15005
+        - containerPort: 15007
         args:
         - proxy
         - pilot
         - -v
         - "2"
         - --discoveryAddress
-        - istio-pilot:15003
+        - istio-pilot:15007
```
Comment (andraxylia) [inline install/kubernetes/templates/istio-pilot.yaml.tmpl]: This change should have been advertised.

**Pull request merged**